src/main/java/com/exasol/adapter/dynamodb/documentfetcher/dynamodb/AbstractDynamodbDocumentFetcher.java

```diff
@@ -20,8 +22,12 @@
 
```

Comment:
I think you should setup your IDE to remind you to change UID for classes you changed. Check also all other classes you touched